fdbserver/TLogServer.actor.cpp

```diff
@@ -84,7 +89,7 @@ struct AlternativeTLogQueueEntryRef {
 		for (auto& msg : *alternativeMessages) {
 			ar.serializeBytes(msg.message);
 		}
-		serializer(ar, knownCommittedVersion, id);
+		serializer(ar, knownCommittedVersion, prevVersion, tLogLocIds, id);
```

Comment:
as above

Reply:
We will need to have a better explanation for this change too.